hack/verify-archives.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ $* != *-repo-root* ]]
```

Comment:
Nit: if there is just 1 required argument, it's typical to just pass it directly as a standalone thing instead of with a flag+thing. E.g.,

```
myscript.sh THING
```

instead of

```
myscript.sh -flag-name=THING
```

Then you can avoid the fancy argument handling in line 23`

Reply:
ack.